### ISSUE TITLE
Accept 0.0.0.0 as a valid host

### ISSUE THREAD

**Issue by drdran** [sukona/Grapevine#41]
Hi!

Putting 0.0.0.0 in the RestServer config, generate an exception on windows 7 64 bit

Regards

**Comment by scottoffen** [comment]
This is a feature of the underlying HttpListener class that Grapevine is build on. You can duplicate it using the following snippet:

```csharp
using System;
using System.Net;

namespace ConsoleApplication1
{
    class Program
    {
        static void Main(string[] args)
        {
            Console.Write("Starting...");

            var listener = new HttpListener();
            listener.Prefixes.Add("http://0.0.0.0:1234/");
            listener.Start();

            Console.WriteLine("done");

            if (listener.Prefixes.Count == 0)
            {
                Console.WriteLine("There are no prefixes.");
            }
            foreach (string prefix in listener.Prefixes)
            {
                Console.WriteLine(prefix);
            }

            Console.ReadLine();
        }
    }
}
```

For the time being, if you would like for Grapevine to listen on all hosts, use the asterisk (*) or the plus sign (+), like so:

```csharp
// using asterisk
var server = new RESTServer { Host = "*" };

// using plus sign
var server = new RESTServer { Host = "+" };
``` 

In future versions, we'll accept `0.0.0.0` and substitute it for one of these being the scenes.

**Comment by drdran** [comment]
Thank you very much Scott, that works like a charm :)

**Comment by drdran** [comment]
Ehm sorry I've close the issue accidentally

**Comment by scottoffen** [comment]
fixed in dev for 4.0
